docs/cvl/methods.md

```diff
@@ -338,16 +338,56 @@ There are several kinds of summaries available:
 (delete-summary)=
 ### Summary application
 
-To decide whether to summarize a given internal or external function call, the
+To decide whether to summarize a given function call at a given call site, the
 Prover first determines whether it matches any of the declarations in the
-methods block, and then uses the declaration and the calling context to
+methods block, and then uses the declaration and the _calling context_ to
```

Comment:
```suggestion
methods block, and afterward uses the declaration and the _calling context_ to
```
(gut feeling)

Reply:
the italics on calling context denote it to be a technical term, but that exact term does not appear again -- if "calling context" is indeed identical with "(3) The context in which it is called", then ideally, the term would be repeated syntactically
(feel free to skip if you think this is overly pedantic)
 --- i'll make a suggestion there ..